cmd/ketch/framework_add.go

```diff
@@ -62,7 +61,8 @@ func newFrameworkAddCmd(cfg config, out io.Writer, addFramework addFrameworkFn)
 }
 
 type frameworkAddOptions struct {
-	name string
+	version string
+	name    string
```

Comment:
can we have a comment that `name` can be either a framework name or a file name?

Reply:
Comment added. I was worried this would be a little unclear, but I don't have a better solution in mind.